History.md

```diff
@@ -1,5 +1,20 @@
 # Liquid Change Log
 
+## 5.1.0 / 2021-03-24
+
+### Fixes
+* Add ParseTreeVisitor to Echo tag (#1414) [CP Clermont]
+* Fix Rubocop (#1402) [Dylan Thacker-Smith]
+* Add a hint to sign CLA only if it's the first time (#1406, #1405) [ADTC]
+* Test with ruby 3.0 as the latest ruby version (#1398) [Dylan Thacker-Smith]
+* Handle carriage return in newlines_to_br (#1391) [Unending]
+
+### Breaking Changes
+* Build the tokenizer through the parse context for liquid-c (#1386) [Dylan Thacker-Smith]
```

Comment:
This just adds a method and changes code to delegate to it, so shouldn't break anything.  Although, thanks for preparing this release, since it is needed if we want to make a liquid-c release with the code currently on its master branch.

Reply:
Moving it to Fixes then and just bumping the minor.